core/src/main/resources/error/error-classes.json

```diff
@@ -82,6 +82,9 @@
     "message" : [ "The fraction of sec must be zero. Valid range is [0, 60]. If necessary set %s to false to bypass this error. " ],
     "sqlState" : "22023"
   },
+  "INVALID_INPUT_INDEX" : {
```

Comment:
Do we need to add a new error? The error message here seems a little inconsistent between map and array access using `[]` operator:
```
SELECT array(1,2,3)[3];   // Do not hint `try_element_at`
SELECT map('a', 1, 'b', 2)['c'];  // Hint `try_element_at`
```